docs/developers-clear-containers-install.md

```diff
@@ -103,7 +129,7 @@ $ ln -s /usr/local/etc/clear-containers/configuration.toml /etc/clear-containers
 
 Edit the file according to your needs.
 
-Refer to [https://github.com/clearcontainers/runtime#global-logging](https://github.com/clearcontainers/runtime#global-logging)
+Refer to [https://github.com/clearcontainers/runtime#debugging](https://github.com/clearcontainers/runtime#debugging)
```

Comment:
Good catch ;)

Aside: I just noticed that the two commands above (`mkdir` and `ln` should be run under `sudo`). Would you mind adding that change to this PR? (In fact the link shouldn't be necessary if the runtime is built as I've descibed above).